packages/tracked-redux/src/-private/tracking.ts

```diff
@@ -12,7 +12,7 @@ const neverEq = (): boolean => false;
 export function createTag(): Tag {
   return createStorage(null, neverEq);
 }
-export const consumeTag = getValue;
```

Comment:
It's also likely that this pattern can defeat some static analysis optimizations, because if the compiler is following the spec strictly it *knows* that that `getValue` can change, but `consumeTag` needs to lock in the current value at this specific point in time, so it won't be able to collapse them down to one thing.